include/tnnt_achievements.h

```diff
@@ -0,0 +1,333 @@
+ACH("Got No Talent", "Scare something by playing music", SCARED_WITH_MUSIC),
+ACH("Cookie Monster", "Read a fortune without eating its cookie", READ_FORTUNE_FOODLESS),
+ACH("Say Hello To My Evil Friend", "Have a non-guaranteed demon lord gated in or summoned", GATED_IN_DEMON_LORD),
+ACH("Make your Mark", "Engrave with an athame", ENGRAVED_WITH_ATHAME),
+ACH("One Hit KO", "Deal a monster its maximum HP or more of damage (minimum 20)", ONE_HIT_KO),
+ACH("Drink It To The Next Level", "Level up by drinking from a sink", LEVELED_UP_FROM_SINK),
+ACH("You're About To Die, You Scrub", "Get a special message saying you're about to die, you scrub", ABOUT_TO_DIE),
+ACH("Extinctionist Lite", "Genocide 20 or more monster species", GENOCIDED_20_SPECIES),
+ACH("Dudley's Hero", "Extinct newts", EXTINCTED_NEWTS),
+ACH("Ice Jesus", "Walk on top of lava unharmed", WALKED_ON_LAVA),
+ACH("Rare Loot", "Acquire an athame that was generated in the inventory of a lich", GOT_LICH_ATHAME),
+ACH("Boy Scout", "Complete Sokoban without incurring any penalties for rule-breaking", DID_PURE_SOKOBAN),
+ACH("Sounds Hollow", "Reveal a secret door with a stethoscope", STETHOSCOPED_SDOOR),
+ACH("Unorthodox Exit", "Reach the Planes via quaffing", PLANES_VIA_QUAFFING),
+ACH("Paddling Pool", "Create pools from a fountain", WATER_GUSH_FORTH),
+ACH("Practical Joke", "Have a tin blow up in your face", BOOBY_TRAPPED_TIN),
+ACH("Amon Hen", "Get either of the two \"vision\" effects from sitting on a throne", VISION_FROM_THRONE),
+ACH("Detonator", "Explode a large box or chest", EXPLODED_LARGE_BOX),
+ACH("Lightbringer", "Light up at least 20 dark rooms in the Dungeons of Doom", LIT_20_ROOMS),
+ACH("Cosmic Bungee Cord", "Get sent back a total of 15 or more levels by the mysterious force", MFORCE_15_LEVELS),
+ACH("No Rulez", "Remove all watchmen from Minetown", GOT_RID_OF_WATCH),
+ACH("Packing Light", "Successfully store a bag of holding inside another bag of holding", NESTED_BOH),
+ACH("Shoo Off", "Make a monster of difficulty 5 or more escape the dungeon", MAKE_MON_ESCAPE),
+ACH("What Have I Unleashed", "Make a demon lord escape the dungeon", MAKE_DLORD_ESCAPE),
+ACH("Walking Bullseye", "Have +30 AC", GOT_30_AC),
+ACH("Doorstopper", "Have a door resist 8 consecutive times when trying to open or close it", DOOR_RESIST_8X),
+ACH("Practically Permanent", "Have more than 10,000 turns of a temporary intrinsic", LONG_TIMEOUT),
+ACH("This One Goes To +8", "Successfully enchant a weapon to +8 or higher", ENCHANT_WEP_TO_8),
+ACH("Sword Plus Plus", "Successfully increase the enchantment on your weapon", ENCHANT_WEAPON),
+ACH("Blasters' Handbook", "Blow up a boulder with a land mine", BLEW_UP_BOULDER),
+ACH("Po-Loew-morph", "Create a golem from a pile of objects", POLY_MADE_GOLEM),
+ACH("Never Tell Me the Odds", "Successfully cast a spell with a failure rate of 95% or greater", CAST_LOCHANCE_SPELL),
+ACH("Merlin's Apprentice", "Successfully cast a level 7 spell", CAST_LV7_SPELL),
```

Comment:
Let's get consensus on some of these new achievement names before merging this. I dislike "Merlin's Apprentice" in particular and also think we can come up with better ones than "Sword Plus Plus" and "Po-Loew-morph" (and "Blasters' Handbook?)"

Reply:
Yep, there is a huge amount of room to improve on those.  I just figured it's better to get something out there for review and change the name, over waiting to implement it until the final name has been decided upon.